### PR TITLE
include the description of one critical Issue 51407 in doc

### PR DESCRIPTION
**Pull request by songrijie**
### What is changed, added or deleted? (Required)

One critical issue 51407 was found which could causes severe issue when prepared plan cache is enabled.  The doc of prepared plan cache was update with the description and workarounds.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-prepared-plan-cache.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ aliases: ['/docs-cn/dev/sql-prepare-plan-cache/','zh/tidb/dev/sql-prepare-plan-c
 
 # Prepare 语句执行计划缓存
 
+> **警告：**
+>
+> 如果已经被缓存的 `Update` 或 `Delete` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致，进而造成结果错误 ([#51407](https://github.com/pingcap/tidb/issues/51407))，请关注修复状态，并升级到 LTS 的最新版本解决这个问题。升级前可能的规避方法：
+> - 执行 DDL 前暂时 [关闭 Prepare 语句的执行计划缓存](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，DDL 执行完毕后再恢复打开。
+> - 避免在业务时间执行 DDL。执行 DDL 之后立即运行 [`admin check table`](/sql-statements/sql-statement-admin-check-table-index.md) 对表和索引的一致性进行验证，一旦发现错误则重建相关索引。
```
Comment (hfxsd) [inline sql-prepared-plan-cache.md:12]:
```suggestion
> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致，进而造成结果错误。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到最新的 LTS 版本解决该问题。升级前可能的规避方法：
> - 执行 DDL 前暂时[关闭 Prepare 语句的执行计划缓存](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，DDL 执行完毕后再恢复打开。
> - 避免在业务时间执行 DDL。执行 DDL 之后立即运行 [`ADMIN CHECK TABLE`](/sql-statements/admin-check-tableindex.md) 对表和索引的一致性进行验证，一旦发现错误则重建相关索引。
```

File: sql-prepared-plan-cache.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ aliases: ['/docs-cn/dev/sql-prepare-plan-cache/','zh/tidb/dev/sql-prepare-plan-c
 
 # Prepare 语句执行计划缓存
 
+> **警告：**
+>
+> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致，进而造成结果错误。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到最新的 LTS 版本解决该问题。升级前可能的规避方法：
+> - 执行 DDL 前暂时[关闭 Prepare 语句的执行计划缓存](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，DDL 执行完毕后再恢复打开。
```
Comment (hfxsd) [inline sql-prepared-plan-cache.md:11]:
```suggestion
> 
> - 执行 DDL 前暂时[关闭 Prepare 语句的执行计划缓存](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，DDL 执行完毕后再恢复打开。
```

File: sql-prepared-plan-cache.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ aliases: ['/docs-cn/dev/sql-prepare-plan-cache/','zh/tidb/dev/sql-prepare-plan-c
 
 # Prepare 语句执行计划缓存
 
+> **警告：**
+>
+> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致，进而造成结果错误。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到最新的 LTS 版本解决该问题。升级前可能的规避方法：
```
Comment (Benjamin2037) [inline sql-prepared-plan-cache.md:10]:
```suggestion
> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到最新的 LTS 版本解决该问题。升级前可能的规避方法：
```

File: sql-prepared-plan-cache.md [hunk 4]
```diff
@@ -5,6 +5,13 @@ aliases: ['/docs-cn/dev/sql-prepare-plan-cache/','zh/tidb/dev/sql-prepare-plan-c
 
 # Prepare 语句执行计划缓存
 
+> **警告：**
+>
+> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到最新的 LTS 版本解决该问题。升级前可能的规避方法：
```
Comment (hfxsd) [inline sql-prepared-plan-cache.md:10]:
```suggestion
> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到[最新的 LTS 版本](https://docs.pingcap.com/zh/tidb/stable)解决该问题。升级前可能的规避方法：
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: sql-prepared-plan-cache.md [hunk 5]
```diff
@@ -5,6 +5,13 @@ aliases: ['/docs-cn/dev/sql-prepare-plan-cache/','zh/tidb/dev/sql-prepare-plan-c
 
 # Prepare 语句执行计划缓存
 
+> **警告：**
+>
+> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行的过程中，同时遇到 DDL 对相关 schema 进行变更，有可能会造成表和索引的数据不一致。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到[最新的 LTS 版本](https://docs.pingcap.com/zh/tidb/stable)解决该问题。升级前可能的规避方法：
```
Comment (Oreoxmt) [inline sql-prepared-plan-cache.md:10]:
```suggestion
> 如果已经被缓存的 `UPDATE` 或 `DELETE` 语句在执行过程中，同时遇到 DDL 对相关 schema 进行变更，可能会导致表和索引的数据不一致。详情参考 [Issue #51407](https://github.com/pingcap/tidb/issues/51407)。请关注该 Issue 的修复状态，并升级到[最新的 LTS 版本](https://docs.pingcap.com/zh/tidb/stable)解决该问题。在升级前，你可以尝试以下规避方法：
```

File: sql-prepared-plan-cache.md [hunk 6]
```diff
@@ -5,6 +5,13 @@ aliases: ['/docs-cn/dev/sql-prepare-plan-cache/','zh/tidb/dev/sql-prepare-plan-c
...
+> 
+> - 执行 DDL 前暂时[关闭 Prepare 语句的执行计划缓存](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，DDL 执行完毕后再恢复打开。
+> - 避免在业务时间执行 DDL。执行 DDL 之后立即运行 [`ADMIN CHECK TABLE`](/sql-statements/sql-statement-admin-check-table-index.md) 验证表和索引的一致性，一旦发现错误则重建相关索引。
```
Comment (Oreoxmt) [inline sql-prepared-plan-cache.md:13]:
```suggestion
> - 在执行 DDL 前，暂时[关闭 Prepare 语句的执行计划缓存](/system-variables.md#tidb_enable_prepared_plan_cache-从-v610-版本开始引入)，DDL 执行完毕后再恢复打开。
> - 避免在业务高峰期执行 DDL。执行 DDL 后立即运行 [`ADMIN CHECK TABLE`](/sql-statements/sql-statement-admin-check-table-index.md) 检查表和索引的一致性，一旦发现错误则重建相关索引。
```

“业务时间”表意不明

**Review by hfxsd — APPROVED** [review]

**Pull request merged**